ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -0,0 +1,203 @@
+      netRegexDe: NetRegexes.startsUsing({ id: '598F', source: 'Nixchen', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '598F', source: 'nixe', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '598F', source: 'ノッケン', capture: false }),
+      alertText: (data, _, output) => output.awayFrom(),
+      outputStrings: {
+        awayFrom: {
+          en: 'Away from Tank and his Tether',
+          ja: 'タンクや線に離れ',
+          cn: '远离坦克及其连线',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Nixie Shower Power',
+      netRegex: NetRegexes.startsUsing({ id: '5991', source: 'Nixie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5991', source: 'Nixchen', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5991', source: 'nixe', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5991', source: 'ノッケン', capture: false }),
+      alertText: (data, _, output) => output.moveTo(),
+      outputStrings: {
+        moveTo: {
+          en: 'Move to ...',
+          ja: '光ってない横列に移動',
+          cn: '站在墙壁未发光的一列',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Nixie Pitter-patter',
+      netRegex: NetRegexes.ability({ id: '5991', source: 'Nixie', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5991', source: 'Nixchen', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5991', source: 'nixe', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5991', source: 'ノッケン', capture: false }),
+      delaySeconds: 3,
+      infoText: (data, _, output) => output.stepIn(),
+      outputStrings: {
+        stepIn: {
+          en: 'Step in Puddle near the Cloud',
+          ja: '雲に近い水を踏む',
+          cn: '站在靠近云朵的水流里等待浮空',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Porxie Tender Loin',
+      netRegex: NetRegexes.startsUsing({ id: '5913', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5913', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5913', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5913', source: 'マザーポークシー', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      delaySeconds: 3,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Matoyas Porxie Huff and Puff',
+      netRegex: NetRegexes.startsUsing({ id: '5919', source: 'Mother Porxie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5919', source: 'Muttersau', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5919', source: 'mère porxie', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5919', source: 'マザーポークシー', capture: false }),
+      alarmText: (data, _, output) => output.standFront(),
+      outputStrings: {
+        standFront: {
+          en: 'Stand in front of Boss (anti-knockback not usable)',
```

Comment:
Maybe something like `get knocked into safe spot`?

Reply:
Sure, but I think it is worth to point out that action cannot be dodged with anti-knockback skill. How is it described in English?

Reply:
Usually, if I'm in raid, I'd say something quick like "goes through knockback invuln" but that's kind of long here. Maybe `no anti-knockback` or `no knockback immunity` is sufficient?